class.jetpack.php

```diff
@@ -610,6 +610,21 @@ private function __construct() {
 		add_action( 'deleted_user', array( $this, 'unlink_user' ), 10, 1 );
 		add_action( 'remove_user_from_blog', array( $this, 'unlink_user' ), 10, 1 );
 
+		// Initialize remote file upload request handlers.
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$is_jetpack_xmlrpc_request = isset( $_GET['for'] ) && 'jetpack' === $_GET['for'] && Constants::get_constant( 'XMLRPC_REQUEST' );
```

Comment:
We've been discussing with @brbrr and it might make sense to remove all these checks, because `wp_ajax_*` hooks will only be called if all those are met anyway, so these checks are redundant?

Reply:
Will follow-up in another PR for this.

Reply:
See #13256